REQUIRE

```diff
@@ -1,4 +1,3 @@
-julia 0.6 0.7-
-MathProgBase 0.5 0.8
+julia 0.7
+MathProgBase 0.7
```

Comment:
The upper bound on MPB is intended and useful. We bump the minor version in cases where downstream packages would break.

Reply:
Ah, I see. Do you have any thoughts on what the appropriate version bounds should be in this case to permit 1.0 support but avoid breakage?

Reply:
See https://github.com/invenia/Convex.jl/pull/3